test/kubernetes_test.go

```diff
@@ -479,8 +840,63 @@ func TestKubernetesIntegrationFallthrough(t *testing.T) {
 	drop 0
     }
 `
-	cases := append(dnsTestCases, dnsTestCasesFallthrough...)
-	doIntegrationTests(t, corefile, cases)
+	doIntegrationTests(t, corefile, dnsTestCasesFallthrough)
+}
+
+func getDynamicEndpoints(Qname, namespace, name string) []dns.RR {
+	rr := []dns.RR{}
+
+	result := strings.Split(getCmdOutput(name, namespace), " ")
+	len_result := len(result)
```

Comment:
I should probably also run
`cmdArgs := kctl + " -n " + name + " get endpoints " + namespace + " -o jsonpath={.subsets[*].addresses[*].ip}"`
to see what this output, 'cause I don't now what happens if you specify a json path